Annotations extras (#13228)
mne-tools/mne-python: doc/changes/devel/13228.newfeature.rst

> Add a ``details`` attribute to :class:`mne.Annotations`, by `Pierre Guetschel`_.

Reviewer: ```suggestion
Add an ``extras`` attribute to :class:`mne.Annotations`, by `Pierre Guetschel`_.
```
